deployment/docker/alerts-health.yml

```diff
@@ -0,0 +1,54 @@
+# File contains default list of alerts for VM components.
+# The alerts below are just recommendations and may require some updates
+# and threshold calibration according to every specific setup.
+groups:
+  - name: vm-health
+    # note the `job` filter and update accordingly to your setup
+    rules:
+      - alert: TooManyRestarts
+        expr: changes(process_start_time_seconds{job=~"victoriametrics|vmselect|vminsert|vmstorage|vmagent|vmalert"}[15m]) > 2
+        labels:
+          severity: critical
+        annotations:
+          summary: "{{ $labels.job }} too many restarts (instance {{ $labels.instance }})"
+          description: "Job {{ $labels.job }} (instance {{ $labels.instance }}) has restarted more than twice in the last 15 minutes.
+            It might be crashlooping."
+
+      - alert: ServiceDown
+        expr: up{job=~"victoriametrics|vmselect|vminsert|vmstorage|vmagent|vmalert"} == 0
+        for: 2m
+        labels:
+          severity: critical
+        annotations:
+          summary: "Service {{ $labels.job }} is down on {{ $labels.instance }}"
+          description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 2 minutes."
+
+      - alert: ProcessNearFDLimits
+        expr: (process_max_fds - process_open_fds) < 100
```

Comment:
These are metrics exported by VM processes (not nodeexporter)

Reply:
this is a very common metric used by various different exporters including kubelet, argo and many more

Reply:
That's fair. I was under the impression this metric name is VM specific.
Anyway, there is no way to get `job` value in alerts - the user can have an arbitrary value. And alerts do not support templating or something like that. So it is up to the user to decide what to alert on:
```
# The alerts below are just recommendations and may require some updates
# and threshold calibration according to every specific setup.
```

Reply:
i was thinking maybe something like we have in health alerts `{job=~"victoriametrics|vmselect|vminsert|vmstorage|vmagent|vmalert"}`

Reply:
That doesn't guarantee the correct selector(
In fact, we need to modify it even for our internal alerts.
So it does either captures all metrics, or captures a fraction or non in case when user does not modify the rule.

Reply:
I propose leaving this alerting rule as as until better solution arises